wp-includes/sqlite/db.php

```diff
@@ -64,10 +64,12 @@
  *
  * TODO: For version 3.0, enforce the DB_NAME constant and remove the fallback.
  */
-if ( defined( 'DB_NAME' ) && '' !== DB_NAME ) {
+if ( defined( 'WP_SQLITE_DB_NAME' ) && '' !== WP_SQLITE_DB_NAME ) {
```

Comment:
I added the prefix `WP_SQLITE_` to be consistent with other constants required by this plugin, such as `WP_SQLITE_AST_DRIVER`.